src/main/kotlin/com/terraformation/backend/accelerator/model/ParticipantProjectForSpecies.kt

```diff
@@ -11,9 +11,11 @@ import com.terraformation.backend.db.default_schema.SpeciesNativeCategory
 import com.terraformation.backend.db.default_schema.tables.references.PROJECTS
 import com.terraformation.backend.db.default_schema.tables.references.SPECIES
 import org.jooq.Record
+import org.jooq.impl.DSL
 
 data class ParticipantProjectsForSpecies(
-    val activeDeliverableId: DeliverableId? = null,
+    // This deliverable ID is associated to the active or most recent cohort module, if available
```

Comment:
If you make this a KDoc comment (put it in `/** */` instead of using `//`) it'll show up as field documentation in the IDE.